doc/source/whatsnew/v0.18.1.rst

```diff
@@ -471,6 +476,11 @@ New Behavior:
 Changes in ``read_csv`` exceptions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. ipython:: python
+   :suppress:
+
+   from io import StringIO
```

Comment:
can you use `import io` instead, and have the import visible in the next block, where it's used.